src/figuresClasses.ts

```diff
@@ -1,19 +1,95 @@
 export interface Figure {
-
+  shape: string;
+  color: string;
+  getArea(): number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape = 'triangle';
+
+  color: string;
+
+  a: number;
+
+  b: number;
+
+  c: number;
```

Comment:
```suggestion
  shape = 'triangle';
  color: string;
  a: number;
  b: number;
  c: number;
```
There is actually no need to add empty lines between each property

Reply:
Linter swears, wants to add empty lines between each property.